starter-code/css/main.css

```diff
@@ -0,0 +1,123 @@
+body {
+  font-family: "Lato", sans-serif;
+  margin: 30px 30px;
+  width: 600px;
```

Comment:
Instead of hardcoding a pixel width try using % so you keep the site responsive.